.github/workflows/cypress_workflow.yml

```diff
@@ -146,70 +203,93 @@ jobs:
       - run: npx cypress cache list
       - run: npx cypress cache path
 
-      # Setup spec files for Dashboards in-house cypress tests
-      - name: Setup spec files for Dashboards tests
-        if: ${{ inputs.specs == '' && matrix.group > 9 }}
+      # Run tests based on configuration
+      - name: Run FT repo tests with no query enhancements
```

Comment:
nit: historically we called this FTRepo.

and I think the specification `with no query enhancements ` might be unnecessary. we should go with the actual one run the query enhancements to say `with query enhancements` incase we want to follow this format in the future and add different type of situations.